syntax = "proto3";

package payload;

message GCS_TO_GSE_STATE_CMD {
    optional bool manual_purge_activate = 1;
    optional bool o2_fill_activate = 2;
    optional bool selector_switch_neutral_position = 3;
    optional bool n20_fill_activate = 4;
    optional bool ignition_fire = 5;
    optional bool ignition_selected = 6;
    optional bool gas_fill_selected = 7;
    optional bool system_activate = 8;

    optional bool manual_purge_activate_inverted = 9;
    optional bool o2_fill_activate_inverted = 10;
    optional bool selector_switch_neutral_position_inverted = 11;
    optional bool n20_fill_activate_inverted = 12;
    optional bool ignition_fire_inverted = 13;
    optional bool ignition_selected_inverted = 14;
    optional bool gas_fill_selected_inverted = 15;
    optional bool system_activate_inverted = 16;
}